response.go

```diff
@@ -125,6 +126,46 @@ func ResponderFromResponse(resp *http.Response) Responder {
 	}
 }
 
+// ResponderFromMultipleResponses wraps an *http.Response list in a Responder.
+//
+// Each response will be returned in the order of the provided list.
+// If the responder is called more than the size of the provided list, an error
+// will be thrown.
+//
+// Be careful, except for responses generated by httpmock
+// (NewStringResponse and NewBytesResponse functions) for which there
+// is no problems, it is the caller responsibility to ensure the
+// response body can be read several times and concurrently if needed,
+// as it is shared among all Responder returned responses.
+//
+// For home-made responses, NewRespBodyFromString and
+// NewRespBodyFromBytes functions can be used to produce response
+// bodies that can be read several times and concurrently.
+func ResponderFromMultipleResponses(responses []*http.Response) Responder {
```

Comment:
We could introduce an optional `fn` parameter like `Times()` and `Once()` have, to allow the user to log when all responders have been called:
```suggestion
func ResponderFromMultipleResponses(responses []*http.Response, fn ...func(...interface{})) Responder {
```

Reply:
agreed, updating the PR in consequence